conf/application.conf

```diff
@@ -18,7 +18,7 @@ appName = "help-to-save-frontend"
 play.http.router = prod.Routes
 
 # to learn why this was included: /display/TEC/2016/03/14/Setting+Security+Headers+in+frontend+services
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:9250 www.google-analytics.com www.youtube-nocookie.com data:; img-src 'self' www.google-analytics.com *.googleusercontent.com "
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:9250 www.google-analytics.com www.youtube.com www.youtube-nocookie.com *.optimizely.com data:; img-src 'self' www.google-analytics.com *.googleusercontent.com img.youtube.com i3.ytimg.com"
```

Comment:
we need to make sure we update `app-config-production` as well before deploying this to prod